src/audio/external.c

```diff
@@ -5526,38 +5527,23 @@ void func_800CA008(u8 arg0, u8 arg1) {
 GLOBAL_ASM("asm/non_matchings/audio/external/func_800CA008.s")
 #endif
 
-#ifdef MIPS_TO_C
-//generated by mips_to_c commit 3c3b0cede1a99430bfd3edf8d385802b94f91307
-extern s8 D_800EA108;
-
 void func_800CA0A0(void) {
+    s32 pad;
+    if (pad){};
```

Comment:
Can we just do:
`if (pad);` with no `{}`?

Reply:
Doesn't appear to match if  you do that. Without the `{};` the `if` statement will implicitly wrap around the actual meat of the function (since its a single line).

So we have to have the `{};` to make the `if` statement truly empty. I don't really understand why the compiler bothers to create stack space if its not going to bother including an actual break in the final asm, but it does so we have to have the `if` statement.